x-pack/plugins/security_solution/public/resolver/store/data/reducer.test.ts

```diff
@@ -59,6 +59,7 @@ describe('Resolver Data Middleware', () => {
     let firstChildNodeInTree: TreeNode;
     let eventStatsForFirstChildNode: { total: number; byCategory: Record<string, number> };
     let categoryToOverCount: string;
+    let aggregateCategoryTotalForFirstChildNode: number;
```

Comment:
:information_source: It's referred to elsewhere as "aggregateTotal" so I kept it here, but I considered another name like "expandedTotal".